test/integration/volume/attach_detach_test.go

```diff
@@ -100,6 +100,9 @@ func TestPodDeletionWithDswp(t *testing.T) {
 	pod := fakePodWithVol(namespaceName)
 	podStopCh := make(chan struct{})
 
+	go informers.Core().V1().PersistentVolumeClaims().Informer().Run(podStopCh)
```

Comment:
Double check which informers should be stopped when podStopCh is closed. Is this correct?